lib/matplotlib/tests/test_colors.py

```diff
@@ -156,6 +156,193 @@ def test_LogNorm():
     assert_array_equal(ln([1, 6]), [0, 1.0])
 
 
+def test_FuncNorm():
```

Comment:
I have now included a test class for each of the implemented classes, and each assert in a separate name method. It also includes a parametric test to replace the loop I had before.